src/App.js

```diff
@@ -90,8 +90,10 @@ const theme = createMuiTheme({
   }
 });
 
+const githubURL = 'https://raw.githubusercontent.com/FSHSchool/FSHOnline-Examples/main/';
```

Comment:
The examples are still in progress. I'll make an addition to the index generation script that will remove directories without children. Also each addition to the Examples repo will be reviewed prior to being merged in, so I imagine quirks like this will be ironed out in the review process.